src/PowerShellEditorServices/Symbols/PesterDocumentSymbolProvider.cs

```diff
@@ -27,63 +27,103 @@ public class PesterDocumentSymbolProvider : FeatureProviderBase, IDocumentSymbol
                 return Enumerable.Empty<SymbolReference>();
             }
 
-            var commandAsts = scriptFile.ScriptAst.FindAll(ast =>
-            {
-            CommandAst commandAst = ast as CommandAst;
+            // Find plausible Pester commands
+            IEnumerable<Ast> commandAsts = scriptFile.ScriptAst.FindAll(IsNamedCommandWithArguments, true);
+
+            return commandAsts.OfType<CommandAst>()
+                              .Where(IsPesterCommand)
+                              .Select(ast => ConvertPesterAstToSymbolReference(scriptFile, ast))
+                              .Where(pesterSymbol => pesterSymbol?.TestName != null);
+        }
+
+        /// <summary>
+        /// Test if the given Ast is a regular CommandAst with arguments
+        /// </summary>
+        /// <param name="ast">the PowerShell Ast to test</param>
+        /// <returns>true if the Ast represents a PowerShell command with arguments, false otherwise</returns>
+        private static bool IsNamedCommandWithArguments(Ast ast)
+        {
 
                 return
-                    commandAst != null &&
+                    ast is CommandAst commandAst &&
```

Comment:
Can ask ever been null on accident? This one I'm pretty sure is no but figured I'd ask.